packages/vanilla/src/components/tabs.ts

```
@@ -1,102 +1,53 @@
+    const scrollLeft = tabList.scrollLeft;
+    const scrollWidth = tabList.scrollWidth;
+    const clientWidth = tabList.clientWidth;
+
+    arrowPrev.classList.toggle('hidden', scrollLeft <= 0);
+    arrowNext.classList.toggle('hidden', scrollWidth <= scrollLeft + clientWidth);
+  }
 
-  const styleTabAttr = document.createElement('style')
-  styleTabAttr.innerText = styledTabs
+  const scrollTabs = (direction: number): void => {
+    tabList.scrollBy({ left: direction * scrollAmount, behavior: 'smooth' });
```

ESLint

[error] Extra semicolon.

(semi)